src/server/setupTabletopAudio.ts

```diff
@@ -9,7 +9,7 @@ import {
   isTabletopAudioAsset,
   isTabletopAudioIndex,
   TabletopAudioIndex,
-} from "../shared/tabletopaudio";
+} from "../shared/tabletopAudio";
 import { batchActions } from "redux-batched-actions";
 import envPaths from "env-paths";
 import lockfile from "proper-lockfile";
```

Comment:
Is it a problem if I rename the `CONTENT_FILE` etc, defined on line `25`? Do they names have any significance?

Reply:
You can rename them 👍